addons/mail/static/src/model/model_field.js

```diff
@@ -792,97 +656,81 @@ class ModelField {
      *
      * @private
      * @param {mail.model} record
-     * @param {string|mail.model|<string|mail.model>[]|null} newValue
+     * @param {mail.model|mail.model[]|null} newValue
      */
     _setRelationUnlinkX2Many(record, newValue) {
-        // convert newValue to array of localId, null is considered unlink all
-        const otherLocalIds = newValue === null
-            ? [...this.read(record)]
+        // null is considered unlink all
+        const deleteOtherRecords = newValue === null
```

Comment:
weird naming for variable holding data, to start with a verb.

Reply:
What is the "delete" equivalent of `newOtherRecords`?
Change both to `toDeleteOtherRecords` and `toAddOtherRecords`?

Reply:
These names are better. To bad the "to X" in naming, but there's honestly no well-defined adjectives to describe these categories... Or maybe there is, but if that's the case there are many consistencies across the code.

Sometimes use of "previous" to mean previous value, while "old" to mean previous values that are no more in use. "new" has been used for values to set, regardless of newly introduced one or existing one... Could be my wordings that are wrong, but I tend to use "new" for non-existing one, so in the context of `model_field.js`, this is misused.